cpp/sarmata-client/wave-utils.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __WAVE_UTILS_H__
+#define __WAVE_UTILS_H__
+
+
+#include <string>
+
+
+/// WAVE file header structure
```

Comment:
It's important here to add information that this software does not handle any header extensions.

Reply:
Added a comment in the file. Also, an exception is thrown [here](https://github.com/techmo-pl/sarmata-client/blob/1-update-api/cpp/sarmata-client/wave-utils.cpp#L14) to signalize that.

Reply:
great